### PR TITLE
add hackerrank tasks, day 10

### PR DESCRIPTION
**Pull request by Korsh**

### CODE REVIEW

File: homeworks/src/andriiarzhanov/com/hackerrank/Day4_Class_vs_Instance/Day4_Class_vs_Instance.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import java.util.Scanner;
+
+/**
+ * Created by arzhanov on 19.04.17.
+ */
+public class Day4_Class_vs_Instance {
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        int T = sc.nextInt();
```
Comment (PBaranovskyi) [inline homeworks/src/andriiarzhanov/com/hackerrank/Day4_Class_vs_Instance/Day4_Class_vs_Instance.java:12]: Did you check it on hackerrank?

Comment (Korsh) [reply homeworks/src/andriiarzhanov/com/hackerrank/Day4_Class_vs_Instance/Day4_Class_vs_Instance.java:12]: This part is default code from task. I copied all code from task just for correct work of project
image

File: homeworks/src/andriiarzhanov/com/hackerrank/Day6_Lets_Review/Day6_Lets_Review.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ * Created by arzhanov on 19.04.17.
+ */
+public class Day6_Lets_Review {
+    public static void main(String[] args) {
+        /* Enter your code here. Read input from STDIN. Print output to STDOUT. Your class should be named Solution. */
+        Scanner scan = new Scanner(System.in);
+        scan.nextInt();
+
+        while (scan.hasNext()) {
+            String stringVar = "", even = "", odd = "";
```
Comment (PBaranovskyi) [inline homeworks/src/andriiarzhanov/com/hackerrank/Day6_Lets_Review/Day6_Lets_Review.java]: Make each in new line

File: homeworks/src/andriiarzhanov/com/hackerrank/Day7_Arrays/Day7_Arrays.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+        Scanner in = new Scanner(System.in);
+        int n = in.nextInt();
+        int[] arr = new int[n];
+        String stringVar = "";
+        for (int i = 0; i < n; i++) {
+            arr[i] = in.nextInt();
+        }
+
+        for (int k = n - 1; k >= 0; k--) {
+            stringVar = stringVar + arr[k] + " ";
```
Comment (PBaranovskyi) [inline homeworks/src/andriiarzhanov/com/hackerrank/Day7_Arrays/Day7_Arrays.java]: You don`t actually need this stringVar variable - you can print straight in for loop

**Review by PBaranovskyi — CHANGES_REQUESTED** [review]

File: homeworks/src/andriiarzhanov/com/hackerrank/Day7_Arrays/Day7_Arrays.java (hunk 3 above)
Comment (Korsh) [reply homeworks/src/andriiarzhanov/com/hackerrank/Day7_Arrays/Day7_Arrays.java]: done

File: homeworks/src/andriiarzhanov/com/hackerrank/Day6_Lets_Review/Day6_Lets_Review.java (hunk 2 above)
Comment (Korsh) [reply homeworks/src/andriiarzhanov/com/hackerrank/Day6_Lets_Review/Day6_Lets_Review.java]: done

**Pull request merged**